ui-tests/src/test/java/io/syndesis/qe/pages/java_interfaces/wizard/WizardCancelable.java

```diff
@@ -0,0 +1,5 @@
+package io.syndesis.qe.pages.java_interfaces.wizard;
+
+public interface WizardCancelable {
```

Comment:
Those 3 interfaces with one method defined seems like an anti-pattern to me? @lvydra wdyt?

Reply:
I'm not sure if it is necessary. It seems to me that "finnishability" is special type of "succeedabilty" and every wizard should be probably "cancelable" in some way, so what about one parent class with finish() (next is finish with local meaning) and cancel() method.

Reply:
Created 1 interface WizardStep - there should be goToPreviousWizardStep() in the future I suppose. The question is where to place the interface. But we can solve it in different PR later for this one not to be blocked longer than necessary. Do you agree?

Reply:
Other common wizard methods can be also added later.